README.md

```diff
@@ -110,7 +115,7 @@ If you run `yarn hardhat --help` you'll get an output of all the tasks you can r
 ## Deploying Contracts
 
 ```
-yarn hardhat deploy
+yarn deploy
```

Comment:
Should we update everything to be npm then if we are telling them to install with npm?

Reply:
Well, npm and npx anyways